warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__trip_planner_rt.sql

```diff
@@ -14,15 +24,24 @@ int_gtfs_quality__trip_planner_rt AS (
         idx.service_key,
         {{ trip_planner_rt() }} AS check,
         {{ compliance_rt() }} AS feature,
-        CASE manual_check__gtfs_realtime_data_ingested_in_trip_planner
-            WHEN 'Yes' THEN {{ guidelines_pass_status() }}
-            WHEN 'No' THEN {{ guidelines_fail_status() }}
-            WHEN 'N/A - no fixed-route service' THEN {{ guidelines_na_check_status() }}
+        CASE
+            -- If there are no RT feeds listed, the service should not fail this check, instead it should be "N/A"
+            -- Note that we're only looking at VP and TU, since that's what it being checked for in this manual check.
+            WHEN tu.status = 'FAIL' AND vp.status = 'FAIL' THEN {{ guidelines_na_check_status() }}
```

Comment:
I think that this will be improved by the indexing changes I am making but it is fine for now (I may end up refactoring a bit but don't think there's a better thing to be done in the meantime short of those larger changes, if that makes sense)